content/rs/administering/cluster-operations/maintenance-mode.md

```diff
@@ -0,0 +1,87 @@
+---
+Title: Maintenance Mode for Cluster Nodes
+description: Prepare node for maintenance, and restore it to previous state
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+When you need to do hardware or operating system maintenance on an RS server,
+it is important that you move all of the shards on that server to another node to avoid the risk of data loss.
+You can use maintenance mode to handle this process simply and efficiently.
+
+## Turning Maintenance Mode ON
+
+When you turn maintenance mode on, RS:
+
+1. Checks whether shutdown of the node causes quorum loss. If so, maintenance mode is not turned on.
+1. Takes a snapshot of the node configuration as a record of which shards and endpoints are on node at that time.
+1. Marks the node as a quorum node to prevent shards from migrating into the node.
```

Comment:
Marks the node as a quorum node to prevent shards and endpoints from migrating into the node.